src/renderer/components/attachment/menuAttachment.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import {
+  Button,
+  Position,
+  Popover,
+  Menu,
+  MenuItem,
+  IconName,
+} from '@blueprintjs/core'
+import { useTranslationFunction } from '../../contexts'
+import { runtime } from '../../runtime'
+
+//function to populate Menu
+const MenuAttachmentItems = ({
+  itemsArray,
+}: {
+  itemsArray: MenuAttachmentItemObject[]
+}) => {
+  const tx = useTranslationFunction()
+  return (
+    <>
+      {itemsArray.map((item: MenuAttachmentItemObject) => (
+        <MenuItem
+          key={item.id}
+          icon={item.icon}
+          text={item.text}
+          onClick={item.onClick}
+          aria-label={tx(item.attachment)}
+        />
+      ))}
+    </>
+  )
+}
+//main component that creates the menu and popover
+const MenuAttachment = ({
+  addFileToDraft,
+}: {
+  addFileToDraft: (file: string) => void
+}) => {
+  const addFilenameFile = async () => {
+    //function for files
+    const file = await runtime.showOpenFileDialog({
+      filters: [
+        {
+          name: 'Files',
+          extensions: [
+            'doc',
+            'docx',
+            'xls',
+            'xlsx',
+            'ppt',
+            'ppt',
+            'pdf',
+            'txt',
+            'csv',
+            'log',
+            'zip',
```

Comment:
I think we shouldn't filter Files by extension, but show all of them. Otherwise it wouldn't be possible to send for example a .db file.

Reply:
also we should be able to send also media files as files, so that way there is still a way to send them without core compression